common/src/value_tree.rs

```diff
@@ -281,6 +301,106 @@ impl ValueView {
     }
 }
 
+#[deprecated(note = "will be removed soon")]
+#[derive(Default)]
+pub struct ValueTreeGasHandler;
+
+impl DAGBasedLedger for ValueTreeGasHandler {
```

Comment:
this stuff now stays only in test code ?

why not move it to the place it is used?

Reply:
Based on our latest understanding the entire `common` crate will sooner or later go, so I just didn't bother moving it anywhere. This specific part is only needed until we've confirmed that the old gas handler implementation (which we believe is correct) produces the identical outcome with the new gas handler (pallet-babsed) for any contract execution given the same chain state. Once confirmed, it won't be needed at all.